rpki-validator/src/main/java/net/ripe/rpki/validator3/api/ApiError.java

```diff
@@ -38,9 +39,12 @@
 @Builder
 @ApiModel(value = "Error")
 public class ApiError {
+    @ApiModelProperty(value = "(HTTP) status code", example = "500", required = true)
     String status;
     String code;
+    @ApiModelProperty(value = "Description of the error", example = "Internal Server Error", required = true)
     String title;
+    @ApiModelProperty(value = "Detailed description of error", required = false)
```

Comment:
It's either "the error" or "error" in both case? I still don't good intuition about it, but I think it would be more consistent.

Reply:
Well caught. I think it is "the error". I rephrased these two comments.